### ISSUE TITLE
Typo on ockam.io

### ISSUE THREAD

**Issue by wbadart** [build-trust/ockam#3707]
Hey team- just noticed the [Products section of ockam.io](https://www.ockam.io/#products) describes the open source offering with

> Scale: Maually configurable
>
> <img width="478" alt="Screen Shot 2022-10-21 at 11 11 57" src="https://user-images.githubusercontent.com/8717474/197261885-8ace449f-504f-4603-8cad-e1cfbfff99cb.png">

I assume this was meant to be "Manually configurable".

Apologies if this is the wrong reporting channel, wasn't sure how better to get in touch.

Looking forward to trying the tool!

**Comment by mrinalwadhwa** [comment]
@wbadart thank you so much for reporting this! We appreciate it a lot!

If you would like to contribute a fix to the text we would happily take it https://github.com/build-trust/ockam-website/blob/develop/src/views/homepage/Packages.tsx#L36

**Comment by wbadart** [comment]
@mrinalwadhwa thanks for the opportunity to contribute! Just opened build-trust/ockam-website#642.

**Comment by mrinalwadhwa** [comment]
Awesome!!

One more quick thing before we can merge please:

- [x] Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository.

The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will make the commit as verified.

**Comment by wbadart** [comment]
build-trust/ockam-contributors#139

**Comment by mrinalwadhwa** [comment]
The fix is deployed https://www.ockam.io/
Thank you @wbadart 🎉

**Comment by wbadart** [comment]
Quick turnaround!
